### PR TITLE
Do not skip tests for IstanbulVM

### PR DESCRIPTION
**Pull request by cburgdorf**
### What was wrong?

We aren't testing Trinity against Istanbul yet.

### How was it fixed?

- This builds upon #1403 
- Wire tests up in tox/circle
- Remove skipping of Istanbul tests

### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [x] Clean up commit history

[//]: # (For important changes that should go into the release notes please add a newsfragment file as explained here: https://github.com/ethereum/trinity/blob/master/newsfragments/README.md)

[//]: # (See: https://trinity-client.readthedocs.io/en/latest/contributing.html#pull-requests)
- [x] Add entry to the [release notes](https://github.com/ethereum/trinity/blob/master/newsfragments/README.md)

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by veox** [comment]
`randomStatetest94` is OK to be tagged `SLOW`, IMO:

https://github.com/ethereum/py-evm/pull/1852#issuecomment-533989714

(It is slow, true, but it crashes the CI box because it consumes a lot of memory.)

**Comment by cburgdorf** [comment]
@veox Thanks for chiming in!

>In fact, I wonder if `SLOWEST_TESTS` should just be imported from py-evm

Generally a good idea I'd say but I'm not sure if it's possible as is because the tests aren't real modules (as in, they don't have a `__init__.py` file) so we can't import from them and if I recall correctly there is some obscure pytest reason for that.

**Comment by veox** [comment]
> ... so we can't import from them ...

Argh, I keep forgetting.

File: tox.ini [hunk 1]
```diff
@@ -3,7 +3,7 @@ envlist=
     py{36,37}-{eth1-core,p2p,p2p-trio,eth1-monitor-trio,integration,lightchain_integration,eth2-core,eth2-fixtures,eth2-integration,eth1-components,eth2-utils}
     py36-long_run_integration
     py36-rpc-blockchain
-    py36-rpc-state-{frontier,homestead,tangerine_whistle,spurious_dragon,byzantium,constantinople,petersburg}
+    py36-rpc-state-{frontier,homestead,tangerine_whistle,spurious_dragon,byzantium,constantinople,petersburg, istanbul}
```
Comment (veox) [inline tox.ini]: Nitpick: extra space.

Comment (cburgdorf) [reply tox.ini]: :+1: fixed

**Pull request merged**